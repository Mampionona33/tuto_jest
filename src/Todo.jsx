import React from 'react';

const Todo = ({ todos }) => {
  return (
    <>
      {todos.map((index, todo) => (
        <>
          <h3>{todo.title}</h3>
          <div>{todo.description}</div>
        </>
      ))}
    </>
  );
};

export default Todo;
