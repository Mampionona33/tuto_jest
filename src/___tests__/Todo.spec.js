import reactTestRenderer from 'react-test-renderer';
import Todo from '../Todo';

test('Todo - should create snapshot', () => {
  const component = reactTestRenderer.create(
    <Todo
      todos={[
        { title: 'item1', description: 'des item 1' },
        { title: 'item2', description: 'desc item 2' },
      ]}
    />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
