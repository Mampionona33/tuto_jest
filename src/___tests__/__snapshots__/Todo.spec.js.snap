// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo - should create snapshot 1`] = `
Array [
  <h3 />,
  <div />,
  <h3 />,
  <div />,
]
`;
